packages/tiny-agents/src/cli.ts

```diff
@@ -4,6 +4,7 @@ import * as readline from "node:readline/promises";
 import { stdin, stdout } from "node:process";
 import { z } from "zod";
 import { PROVIDERS_OR_POLICIES } from "@huggingface/inference";
+import type { ServerConfig } from "@huggingface/mcp-client";
```

Comment:
a bit confusing to have the same name for something different in mcp-client vs. tiny-agent, but I guess this one's on me 😀

i guess we could potentially rename the one in mcp-client to McpServerConfig but 🤷